pkgs/applications/version-management/git-credential-manager/default.nix

```diff
@@ -7,38 +7,49 @@
 , libSM
 , fontconfig
 , libsecret
+, git
+, git-credential-manager
 , gnupg
 , pass
+, testers
 , withGuiSupport ? true
 , withLibsecretSupport ? true
 , withGpgSupport ? true
 }:
 
+# note: requires sandbox to be disabled on MacOS, so that
+# .NET can find `/usr/bin/codesign` to sign the compiled binary
```

Comment:
This should probably be added in the meta.longDescription attribute so users can actually find it since most people won't look at a package definition.

Reply:
This sounds like it's a general issue on macOS affecting many packages but I'm personally not using macOS, so I can't check. Should we really add this comment to specifically this package?

Reply:
For me it sounds like a specific issue with `/usr/bin/codesign` on darwin so I would still add it to this package. But I don't even own a mac so I don't care too much tbh. And I think we can both agree that meta.longDescription is easier to find for users getting having this problem then some comment in the package definition?

Reply:
`git-credential-manager` can't be the only package that requires code signing on macOS, especially as this isn't explicitly done anywhere. But yes, the `longDescription` would be a better place than a comment if we have to add it here.

Reply:
I'm in favor of adding it here so we can avoid user frustration, because it doesn't really hurt if we have it and in case it's not that probably not that unlikely that this is the  first time $user needs codesign.

Reply:
As far as I can tell, upstream is only doing codesigning in the GitHub action. I have the sandbox set to relaxed on Darwin, and this builds, but it crashes because it’s not signed at all. It should be possible to use darwin.sigtool’s `codesign` in a `postFixup` to manually ad hoc sign the resulting binary. 

Also, Is it possible to use `install_name_tool` to make it use libc++ from nixpkgs instead of the system one? I assume that’s an artifact of creating a self-contained binary?